splunk-otel-android/src/test/java/com/splunk/rum/SplunkRumTest.java

```diff
@@ -59,13 +60,10 @@
 public class SplunkRumTest {
 
     @Rule public OpenTelemetryRule otelTesting = OpenTelemetryRule.create();
+
     private Tracer tracer;
-    private final Config config =
-            Config.builder()
-                    .realm("us0")
-                    .rumAccessToken("token")
-                    .applicationName("appName")
-                    .build();
+    private final AtomicReference<Attributes> globalAttributes =
+            new AtomicReference<>(Attributes.empty());
```

Comment:
I like to have a least one random attribute in test data like this to help ward off the case where the empty instance is being used on accident.

Reply:
Done. I had to create a separate attributes ref for the location tests though.